File: Hima Rani Mathews/if-else/Absolute.cpp
#include<iostream>
using namespace std;
int main(){
	int a;
	cout<<"\nEnter the integer : ";
	cin>>a;
	if(a>0){
		cout<<"Absolute Value of "<<a<<" is "<<a;
	}
	else{
		cout<<"Absolute Value of "<<a<<" is "<<(a*-1);
	}
}

File: Arnav Sharma/Assignment-0/if-else/absolute.java
public class absolute {
    static void findAbsolute(int N)
    {
        if (N < 0)
        {
            N = (-1) * N;
        }

        System.out.printf("%d ", N);
    }

    public static void main(String[] args)
    {

        int N = -12;
        findAbsolute(N);
    }
}
